### PR TITLE
chore(suite): copy change in container when invalid passphrase

### LINKED ISSUE #7557: Unify invalid passphrase errors to passphrase mismatch
Passphrase mismatch language is misleading. 

Screenshot 2023-02-06 at 16 57 07
Screenshot 2023-02-06 at 16 55 37

### PR DESCRIPTION
**Pull request by AdamSchinzel**
Dropping `TR_AUTH_CONFIRM_FAILED_DESC`

## Description

Dropped `TR_AUTH_CONFIRM_FAILED_DESC` from the dashboard portfolio card when something failed on invalid passphrase.

## Related Issue

Resolve #7557

### CODE REVIEW

File: packages/suite/src/views/dashboard/components/PortfolioCard/components/Exception.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ interface CTA {
 
 interface ContainerProps {
     title: ComponentProps<typeof Translation>['id'];
-    description: ComponentProps<typeof Translation>['id'] | JSX.Element;
+    description?: ComponentProps<typeof Translation>['id'] | JSX.Element;
```
Comment (komret) [inline packages/suite/src/views/dashboard/components/PortfolioCard/components/Exception.tsx]: `ComponentProps<typeof Translation>['id']` could be simplified to `TranslationKey`.

**Review by komret — APPROVED** [review]
Approving, suggesting optional small code improvement.

File: packages/suite/src/views/dashboard/components/PortfolioCard/components/Exception.tsx (hunk 1 above)
Comment (AdamSchinzel) [reply packages/suite/src/views/dashboard/components/PortfolioCard/components/Exception.tsx]: c18a00c7547454896a6d6a4726686297dd059d40, also found some occurrences in one other file

**Comment by bosomt** [comment]
**QA OK**

passphrase mismatch result:

**toast**: Passphrase confirmation error: Invalid passphrase
**dashboard**: Wrong passphrase

**Pull request merged**